packages/react-checkbox/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -4,6 +4,7 @@ import { create } from '@uifabric/utilities/lib/test';
 import { mount, ReactWrapper } from 'enzyme';
 
 import { Checkbox } from './Checkbox';
+import { isConformant } from '../../common/isConformant';
```

Comment:
Can you add this for the /next checkbox too?

Same for link, slider, tabs, and toggle's /next versions.

Reply:
i decided to not duplicate the tests (yet) in `/next` folders. so no need here

Reply:
Why not? To me it seems good to have the conformance tests start running early. (Though if we did that we'd probably need to add an option to react-conformance to use a different entry point file besides `src/index` for parsing.)

Reply:
i was trying to avoid duplication of the same code in the code base. `/next` is not something we should worry until we get v8 out of the door. we have 2 copies now, didn't want to make it 3.